app/build.gradle

```diff
@@ -152,8 +153,9 @@ dependencies {
     implementation 'com.journeyapps:zxing-android-embedded:3.6.0'
 
     //Navigation
-    implementation 'android.arch.navigation:navigation-fragment-ktx:1.0.0-rc02'
-    implementation 'android.arch.navigation:navigation-ui-ktx:1.0.0-rc02'
+    implementation "androidx.navigation:navigation-fragment-ktx:$nav_version" // For Kotlin use
+    // navigation-fragment-ktx
+    implementation "androidx.navigation:navigation-ui-ktx:$nav_version"
```

Comment:
Why these changes?

Reply:
migrate to androidX, It was using pre-androidX versions

Reply:
I don't think it relates to issue #1242. You can open a new issue for this.